unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/DCLAudioStream.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace DCL.Components
+{
+    public class DCLAudioStream : BaseComponent
+    {
+        [System.Serializable]
+        public class Model
+        {
+            public string url;
+            public bool playing = false;
+            public float volume = 1;
+        }
+
+        public Model model;
+        private bool isPlaying = false;
+
+        public override IEnumerator ApplyChanges(string newJson)
+        {
+            Model prevModel = model;
+            model = SceneController.i.SafeFromJson<Model>(newJson);
+
+            bool forceUpdate = prevModel.volume != model.volume;
+
+            UpdatePlayingState(forceUpdate);
+
+            yield return null;
+        }
+
+        private void Start()
+        {
+            CommonScriptableObjects.playerCoords.OnChange += OnPlayerCoordsChanged;
+            CommonScriptableObjects.rendererState.OnChange += OnRendererStateChanged;
+        }
+
+        private void OnDestroy()
+        {
+            CommonScriptableObjects.playerCoords.OnChange -= OnPlayerCoordsChanged;
+            CommonScriptableObjects.rendererState.OnChange -= OnRendererStateChanged;
+            StopStreaming();
+        }
+
+        private bool AreCoordsInsideComponentScene(Vector2Int coords)
+        {
+            if (scene == null) return false;
+            return scene.parcels.Contains(coords);
```

Comment:
We can check component's scene id against `SceneController.currentSceneId` instead of this expression.